Adapted from 4f15d4449a31d551d61d0a00362089d8d7d8a763 Mon Sep 17 00:00:00 2001
From: Jean-Yves Avenard <jyavenard@mythtv.org>
Date: Sat, 05 Apr 2014 14:55:04 +0000
Subject: Fix linking issues.

Spotted by Craig Treleaven, freetype linking parameters were added twice and too early, 
causing general linking with installed system libraries first, and then local tree ones.
Configure also automatically adds all libraries checked for to EXTRALIBS define
---
diff -u mythtv/libs/libmythtv/libmythtv.pro.orig mythtv/libs/libmythtv/libmythtv.pro
--- mythtv/libs/libmythtv/libmythtv.pro.orig	2014-04-09 10:50:56.000000000 -0400
+++ mythtv/libs/libmythtv/libmythtv.pro	2014-04-09 10:51:46.000000000 -0400
@@ -50,7 +50,6 @@
 INCLUDEPATH += $$POSTINC
 
 QMAKE_CXXFLAGS += $${FREETYPE_CFLAGS}
-QMAKE_LFLAGS_SHLIB += $${FREETYPE_LIBS}
 
 macx {
     # Mac OS X Frameworks
diff -u mythtv/settings.pro.orig mythtv/settings.pro
--- mythtv/settings.pro.orig	2014-04-09 10:47:39.000000000 -0400
+++ mythtv/settings.pro	2014-04-09 10:54:04.000000000 -0400
@@ -158,7 +158,6 @@
 }
 EXTRA_LIBS  = $$EXTRALIBS
 
-EXTRA_LIBS += $$FREETYPE_LIBS
 EXTRA_LIBS += $$CONFIG_FIREWIRE_LIBS
 
 EXTRA_LIBS += $$LOCAL_LIBDIR_OGL
